packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,27 @@
+import { SecuritySchemeElement } from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement, toValue } from '@swagger-api/apidom-core';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securitySchemeTypeRefractorPlugin =
+  ({ annotations }: SecuritySchemeTypePluginOptions) =>
+  () => ({
+    visitor: {
+      SecuritySchemeElement(element: SecuritySchemeElement) {
+        if (toValue(element.type) !== 'mutualTLS') return undefined;
+
+        const annotation = new AnnotationElement(
+          '"mutualTLS" type is not supported in OpenAPI 3.0.3. It will be removed from the converted document.',
+          { classes: ['error'] },
+          { code: 'mutualTLS' },
+        );
+
+        annotations.push(annotation);
+        return new SecuritySchemeElement();
```

Comment:
`cloneShallow` comes from `@swagger-api/apidom-core`.

```suggestion
        const securitySchemeElementCopy = cloneShallow(element);
        securitySchemeElementCopy.remove('type');
        
        return securitySchemeElementCopy;
```

> Annotate and remove this type/leave empty string

I would assume this means that we want to remove the value of the `SecurityScheme.type` fixed field. But we're removing all fields from the `SecurityScheme` and basically creating and empty Security Scheme Object. If that was the intention then the code is correct. If the intention was to just remove the `type` fixed field, the code needs to look like I've drafter above.